File: account/regional-settings/get-timezone/code-samples/readTimezone.py
# https://developers.ringcentral.com/my-account.html#/applications
# Find your credentials at the above url, set them as environment variables, or enter them below

# PATH PARAMETERS
timezoneId = '<ENTER VALUE>'

# OPTIONAL QUERY PARAMETERS
queryParams = {
    #'page': '1',
    #'perPage': '100'
}

import os
from ringcentral import SDK
rcsdk = SDK(os.environ['clientId'], os.environ['clientSecret'], os.environ['serverURL'])
platform = rcsdk.platform()
platform.login(os.environ['username'], os.environ['extension'], os.environ['password'])
r = platform.get(f'/restapi/v1.0/dictionary/timezone/{timezoneId}', queryParams)

File: account/regional-settings/get-timezone/code-samples/readTimezone.cs
// https://developers.ringcentral.com/my-account.html#/applications
// Find your credentials at the above url, set them as environment variables, or enter them below

// PATH PARAMETERS
string timezoneId = "<ENTER VALUE>";

// OPTIONAL QUERY PARAMETERS
ReadTimezoneParameters readTimezoneParameters = new ReadTimezoneParameters {
    //page = '1',
    //perPage = '100'
};

RestClient rc = new RestClient(
    Environment.GetEnvironmentVariable("clientId"),
    Environment.GetEnvironmentVariable("clientSecret"),
    false
);
await rc.Authorize(
    Environment.GetEnvironmentVariable("username"),
    Environment.GetEnvironmentVariable("extension"),
    Environment.GetEnvironmentVariable("password")
);
var r = await rc.Restapi().Dictionary().Timezone(timezoneId).Get(readTimezoneParameters);
// PROCESS RESPONSE
